tests/test_events.py

```diff
@@ -212,7 +214,7 @@ def test_elasticsearch_connection(events_object, caplog):
         es_mock.return_value = True
         action_thread.join()
         assert "Connection to ES failed. Trying to connect to back..." in caplog.text
-        assert "Stable connection to ES." in caplog.text
+        # assert "Stable connection to ES." in caplog.text
```

Comment:
these logs were added at some point for the purpose of checking autoconnection/disconnection. I'm not sure exactly which ticket it was, but @mariacarmina can help refer that.

If they are no longer needed, can you please add some details why? And remove entirely, instead of commenting them out.

Reply:
Having Aqua output a couple of lines in the log every second
```  Stable connection to ES.  ```  is not the right approach. It just fills the log with unwanted information and makes debug more complicated.

We should log when something is not right, not when everything is good :)

Since I have removed that annoying log, I had to change the test, because the test was looking for that log :)

Reply:
> I had to revert [6ef8945](https://github.com/oceanprotocol/aquarius/pull/928/commits/6ef894566d9fbe42dc457a5615d94c4f89355bf7) (sorry about the typo in commit name) because tests have to be rewritten.
> 
> there is a function called "deploy_datatoken" which actually deploys a NFT :) and in all of our tests, we don't have datatokens and valid ddos, so we are going to rewrite them in a proper way later

Let's create a ticket regarding this :)

Reply:
I wanted to capture the moment when the connection to ES was up, even after an outage. It can be removed

Reply:
then it's settled :)